ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -48,8 +48,18 @@ Column {
 
             property bool loadingFailed: false
 
+            property bool unfurle: true
```

Comment:
unfurle is not a valid word. Do you mean the verb "unfurl" or the adjective "unfurled"?